sakisandrac/closet-tracker-api
# Sc/feature/data

What I did:
- Created 2 GET endpoints for closet data, and outfits data
- Created mock data in the data JS file with the data for one single user
- Created a post endpoint to allow FE to upload data on clothing items
- Installed nodemon so that server automatically refreshes when edited
- Uploaded sample images to use in data file for each category of items

Testing Notes/Extras: 
- run npm i to install dependencies
- start server by run nodemon with the command `nodemon index.js` in terminal
- you can test out the get endpoints in the browser!

Did this break anything?

Type of change


Checklist:


What's next?
- I tested the post and get endpoints, probably would be good to make a post endpoints for outfits data, along with patch and delete endpoints as well.
- Error handling will need to be added. The only error handling i have is a check to make sure all parameters are included in the request when FE is posting data.
